openslides/motions/static/js/motions/site.js

```diff
@@ -1119,6 +1127,26 @@ angular.module('OpenSlidesApp.motions.site', [
         $scope.reset_state = function () {
             $http.put('/rest/motions/motion/' + motion.id + '/set_state/', {});
         };
+        // save additional state field
+        $scope.saveAdditionalStateField = function (stateExtension) {
+            if (stateExtension) {
+                motion["comment " + $scope.commentFieldForState] = stateExtension;
+                motion.title = motion.getTitle(-1);
+                motion.text = motion.getText(-1);
+                motion.reason = motion.getReason(-1);
+                Motion.save(motion);
```

Comment:
Under circumstances this creates a new motion version. Do you want to do so only to set the comment field name for the state or recommendation?